frontend/javascripts/admin/organization/organization_edit_view.tsx

```diff
@@ -33,6 +50,8 @@ class OrganizationEditView extends React.PureComponent<Props, State> {
     pricingPlan: null,
     isFetchingData: false,
     isDeleting: false,
+    organization: null,
+    activeUsersCount: 1,
```

Comment:
I'd initialize this to null to mark this as "is being loaded" to avoid incorrect values being rendered (this doesn't happen in the moment, because of the way things are fetched, but I'd still try to avoid incorrect values when initializing).

Reply:
That would require changing the type signature from `activeuserCount: number` to `activeuserCount: number|null` and checking for `not null` everywhere. :-/